samples/Chat/src/app/ConfigurationScreen.tsx

```diff
@@ -129,10 +150,21 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
           return;
         }
         setConfigurationScreenState(CONFIGURATIONSCREEN_SHOWING_JOIN_CHAT);
+
+        // Join with URL Parameters
+        const threadId = getExistingThreadIdFromURL();
+        const token = getExistingTokenFromURL();
+        const userId = getExistingUserIdFromURL();
+        const displayName = getExistingDisplayNameFromURL();
+        const endpointUrl = getExistingEndpointURLFromURL();
+
+        if (token && userId && displayName && threadId && endpointUrl) {
+          joinChatThreadWithExistingUser(token, userId, displayName, threadId, endpointUrl);
+        }
```

Comment:
Move this line to below `joinChatThreadWithExistingUser` as `joinChatThreadWithExistingUser` is replacing what is set here.

```suggestion
        // Check if we have all the required parameters supplied as query search params.
        const threadId = getExistingThreadIdFromURL();
        const token = getExistingTokenFromURL();
        const userId = getExistingUserIdFromURL();
        const displayName = getExistingDisplayNameFromURL();
        const endpointUrl = getExistingEndpointURLFromURL();

        if (token && userId && displayName && threadId && endpointUrl) {
          joinChatThreadWithExistingUser(token, userId, displayName, threadId, endpointUrl);
          return;
        }
        
        // Else show the join chat screen where a user enters there display name and the other args are collected from the server
        setConfigurationScreenState(CONFIGURATIONSCREEN_SHOWING_JOIN_CHAT);
```